tests/testenv/cli_test_env.go

```diff
@@ -426,10 +426,12 @@ func randomName(opt DirectoryTreeOptions) string {
 	maxNameLength := intOrDefault(opt.MaxNameLength, 15)
 	minNameLength := intOrDefault(opt.MinNameLength, 3)
 
-	b := make([]byte, rand.Intn(maxNameLength-minNameLength)+minNameLength)
+	l := rand.Intn(maxNameLength-minNameLength) + minNameLength
```

Comment:
I think this should be 
```suggestion
	l := rand.Intn(maxNameLength-minNameLength + 1) + minNameLength
```
At least when I think of "MaxNameLength" I'd expect the output possibilities should be inclusive of that value.

Reply:
+1 makes intuitive sense